integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -318,10 +334,10 @@ def test_hash_reduction_decimal_overflow_sum(precision):
         # some optimizations are conspiring against us.
         conf = {'spark.rapids.sql.batchSizeBytes': '128m'})
 
-@pytest.mark.parametrize('data_gen', [_longs_with_nulls], ids=idfn)
-def test_hash_grpby_sum_count_action(data_gen):
+@pytest.mark.parametrize('data_gen', [_grpkey_nested_structs_with_array_basic_child,  _longs_with_nulls] + _grpkey_list_with_non_nested_children, ids=idfn)
```

Comment:
This is not the test you want to modify. It is testing a very specific corner case where we are checking the row counts of a SUM match.  This is an odd case where the result of the aggregation is thrown away.  We should have a separate test that verifies we get the correct answer for these types of groupby keys.


Also how in the world are you doing a SUM on a String?